src/utils/daos_hdlr.c

```diff
@@ -3004,12 +2743,11 @@ dm_connect(struct cmd_args_s *ap,
 	if (daos_handle_is_valid(ca->src_poh)) {
 		rc2 = daos_pool_disconnect(ca->src_poh, NULL);
 		if (rc2 != 0)
-			fprintf(ap->errstream,
-				"failed to disconnect from source pool "DF_UUIDF": "DF_RC"\n",
-				DP_UUID(ca->src_p_uuid),
-				DP_RC(rc2));
+			DH_PERROR_DER(ap, rc2, "Failed to disconnect from source pool "DF_UUIDF,
+			      	      DP_UUID(ca->dst_p_uuid));
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs